ansible/tasks/wifi.yml

```diff
@@ -0,0 +1,11 @@
+
+---
+- name: "install wpa_supplicant.conf"
+  template:
+    src: "templates{{ item }}.j2"
```

Comment:
I think you can drop the `templates`, since that is already assumed by the `template` module.